drivers/SmartThings/zigbee-thermostat/profiles/thermostat-popp-danfoss.yml

```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatHeatingSetpoint
     version: 1
+    config:
+      values:
+        - key: "heatingSetpoint.value"
+          range: [5, 35]
```

Comment:
- I'd add `step: 0.5 ` here
- additionally - this device's temperatureMeasurement capability range is [0 , 40]